h2o-hadoop/tests/python/pyunit_gbm_on_hadoop.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/env/python
+
+import sys, os
+sys.path.insert(1, os.path.join("..","..",".."))
+from tests import pyunit_utils
+import h2o
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+
+
+def gbm_on_hadoop():
+  local_frame = h2o.import_file('https://h2o-public-test-data.s3.amazonaws.com/smalldata/logreg/prostate.csv')
```

Comment:
Note: in multinode case, prostate will fit into a single node, if we go forward with multinode scenarios, we will need to use larger data or force distribute the prostate.

Reply:
Do we need to test multinode?

My point of view is that these kind of tests make sure that we work well with the API. We should still keep a Hadoop cluster to run a multinode distrubuted test. But for the "hadoop distribution" tests I don't think we really need that.

Reply:
good point